src/main/java/org/spongepowered/common/entity/EntityUtil.java

```diff
@@ -1205,6 +1205,17 @@ public static ItemStack throwDropItemAndConstructEvent(IMixinEntity entity, doub
         }
     }
 
+    /**
+     * This is used to create the "dropping" motion for items caused by players. This
+     * specifically was being used (and should be the correct math) to drop from the
+     * player, when we do item stack captures preventing entity items being created.
+     *
+     * @param dropAround True if it's being "dropped around the player like dying"
+     * @param player The player to drop around from
+     * @param random The random instance
+     * @return The motion vector
+     */
+    @SuppressWarnings("unused")
```

Comment:
Why should we keep the unused?

Reply:
Item pre-merging will be reintroduced, just not right now.